docs/analyzer-rules/AZFW0011.md

```diff
@@ -0,0 +1,32 @@
+# AZFW0011: Invalid binding type
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW00011|
+| **Category** |[Usage]|
+| **Severity** |Error|
+
+## Cause
+
+This rule is triggered when a function is binding to a non-iterable type for a  blob container path.
+
+## Rule description
+
+For a function that has a BlobInput attribute with a container path, the parameter must be of iterable type such as `IEnumerable<T>`.
+
+For example, if your function has attribute `BlobInput("container")` and it is binding to a `string`, then you will see this error.
+
+
+## How to fix violations
+
+Change the binding type to an iterable type such as `IEnumerable<T>` or provide blob path `container/blob` instead of container path for non-iterable binding type.
+
+### Supported Types
+
+You can refer to the [public documentation](https://learn.microsoft.com/azure/azure-functions/functions-bindings-storage-blob?tabs=in-process%2Cextensionv5%2Cextensionv3&pivots=programming-language-csharp) to see which types are supported for blob container.
+
+| Binding | Supported Types |
```

Comment:
Since we have the link to the public docs, we should remove the table from here to make sure this never gets out of sync